boards/arm/degu_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,103 @@
+# Kconfig - Degu Evaluation Kit configuration
+#
+# Copyright (c) 2019 Atmark Techno, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DEGU_EVK
+
+config BOARD
+	default "degu_evk"
+
+if USB
+
+config USB_NRF52840
+	def_bool y
+
+config USB_DEVICE_STACK
+	def_bool y
+
+endif # USB
+
+if IEEE802154
+
+config IEEE802154_NRF5
+	def_bool y
+
+endif # IEEE802154
+
+config I2C
+	def_bool y
+
+if I2C
+
+config I2C_NRFX
+	def_bool y
+
+if I2C_NRFX
+
+config I2C_0
+	def_bool y
+
+config I2C_1
+	def_bool y
+
+endif #I2C_NRFX
+
+if I2C_0
+
+choice I2C_0_NRF_TYPE
+	default I2C_0_NRF_TWIM
+
+endchoice
+
+endif # I2C_0
+
+if I2C_1
+
+choice I2C_1_NRF_TYPE
+	default I2C_1_NRF_TWIM
+
+endchoice
+
+endif # I2C_1
+
+endif # I2C
+
+if ADC
+
+config ADC_NRFX_SAADC
+	def_bool y
```

Comment:
This is not needed, as well as the `if` that follows. The `ADC_NRFX_SAADC` option is set by default to `y` for every nRF SoC that has the SAADC peripheral (see [drivers/adc/Kconfig.nrfx](https://github.com/zephyrproject-rtos/zephyr/blob/master/drivers/adc/Kconfig.nrfx#L30)), when `ADC` is enabled, obviously.